python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -463,6 +463,16 @@ def _db_connect(self):
             with contextlib.closing(connection.cursor()) as cursor:
                 yield (connection, cursor)
 
+    def _compute_sys_path(self):
+        """
+        :returns: Path to the current core.
+        """
+        # While core swapping, the Python path is not updated with the new core's Python path,
+        # so make sure the current core is at the front of the Python path for out subprocesses.
+        tank_folder = os.path.dirname(sgtk.__file__)
```

Comment:
debug logging here to include the path that we are adding to sys.path could be helpful for debugging!